src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -296,43 +294,62 @@ class FunSpec private constructor(builder: Builder) {
         = addParameter(name, type.asTypeName(), *modifiers)
 
     fun addCode(format: String, vararg args: Any) = apply {
-      code.add(format, *args)
+      body.add(stripReturn(format), *args)
+    }
+
+    private fun stripReturn(format: String) = when {
```

Comment:
I'm a little concerned about this method because of it's side-effect in setting the builder boolean. Can we call it something like `expressionBodyHandler` instead?